src/cargo/sources/path.rs

```diff
@@ -210,39 +210,43 @@ impl<'cfg> PathSource<'cfg> {
                         self.config.shell().warn(format!(
                             "Pattern matching for Cargo's include/exclude fields is changing and \
                              file `{}` WILL be excluded in a future Cargo version.\n\
-                             See https://github.com/rust-lang/cargo/issues/4268 for more info",
+                             See <https://github.com/rust-lang/cargo/issues/4268> for more\
+                             information.",
                             relative_path.display()
                         ))?;
                     } else {
                         self.config.shell().warn(format!(
                             "Pattern matching for Cargo's include/exclude fields is changing and \
                              file `{}` WILL NOT be included in a future Cargo version.\n\
-                             See https://github.com/rust-lang/cargo/issues/4268 for more info",
+                             See <https://github.com/rust-lang/cargo/issues/4268> for more\
+                             information.",
                             relative_path.display()
                         ))?;
                     }
                 } else if no_include_option {
                     self.config.shell().warn(format!(
                         "Pattern matching for Cargo's include/exclude fields is changing and \
                          file `{}` WILL NOT be excluded in a future Cargo version.\n\
-                         See https://github.com/rust-lang/cargo/issues/4268 for more info",
+                         See <https://github.com/rust-lang/cargo/issues/4268> for more\
+                         information.",
                         relative_path.display()
                     ))?;
                 } else {
                     self.config.shell().warn(format!(
                         "Pattern matching for Cargo's include/exclude fields is changing and \
                          file `{}` WILL be included in a future Cargo version.\n\
-                         See https://github.com/rust-lang/cargo/issues/4268 for more info",
+                         See <https://github.com/rust-lang/cargo/issues/4268> for more\
```

Comment:
I think you need a space before escaping the return to preserve the space break.  Here and in the previous warnings.

Reply:
Yep, good call.